projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/kubernetes/ControlKubernetesService.java

```diff
@@ -21,8 +21,7 @@ public class ControlKubernetesService extends KubernetesService {
   // those should be null/empty when Control service is deployed in k8s hence default is empty
   public ControlKubernetesService(
       @Value("${datajobs.control.k8s.namespace:}") String namespace,
-      @Value("${datajobs.control.k8s.kubeconfig:}") String kubeconfig,
-      @Value("${datajobs.control.k8s.k8sSupportsV1CronJob}") boolean k8sSupportsV1CronJob) {
-    super(namespace, kubeconfig, k8sSupportsV1CronJob, log);
+      @Value("${datajobs.control.k8s.kubeconfig:}") String kubeconfig) {
+    super(namespace, kubeconfig, log);
   }
```

Comment:
the boolean toggle was introduced for backwards & forward compatibility that is configurable. Please describe the compatibility proposed

Reply:
No backwards compatibility supported. 
This changes removes backwards compatility support.

Reply:
Since the users could be running any Kubernetes, isn't it more valuable for the OSS project to additionally cover those who had not upgraded due to some reason, instead of discarding them from using VDK on older Kubernetes? There is this group of users X that may not be able to upgrade, and still want to use VDK. Why are we now making the k8s version a hard requirement all of a sudden?

Reply:
> If we choose to support this beta version VDK we will never be able to upgrade to the latest k8s clients and won't be able to use the latest features k8s has to offer.

Could you please elaborate?